src/ol/coordinate.js

```diff
@@ -415,7 +415,7 @@ export function toStringXY(coordinate, opt_fractionDigits) {
  */
 export function wrapX(coordinate, projection) {
   const projectionExtent = projection.getExtent();
-  if (projection.canWrapX() && (coordinate[0] < projectionExtent[0] || coordinate[0] > projectionExtent[2])) {
+  if (projection.canWrapX() && (coordinate[0] < projectionExtent[0] || coordinate[0] >= projectionExtent[2])) {
```

Comment:
Fine for the extent's center, but here I think we do not want to include the right edge.

Reply:
There is a similar problem with these extents [-360, -90, 0, 90] [0, -90, 360, 90] [360, -90, 720, 90] as with the center.  They should be all produce the same real world extent but [0, -90, 360, 90] doesn't.

Reply:
But `ol/coordinate#wrapX()` is not involved in `ol/extent#wrapX`, and also not in `ol/layer/Graticule`. So I wonder why you insist on this change in `ol/coordinate`.

Reply:
I am fine with your change in `ol/extent#wrapX`, if that was not clear.

Reply:
`ol/coordinate#wrapX` is used in `prepareFrame` where it should be consistent with the extent

Reply:
Good point.